model-optimizer/extensions/back/ChangeRandomUniformOutputType.py

```diff
@@ -0,0 +1,37 @@
+# Copyright (C) 2018-2021 Intel Corporation
+# SPDX-License-Identifier: Apache-2.0
+
+import numpy as np
+
+from extensions.ops.Cast import Cast
+from mo.back.replacement import BackReplacementPattern
+from mo.graph.graph import Graph
+from mo.middle.passes.convert_data_type import data_type_str_to_np
+
+
+class ChangeRandomUniformOutputType(BackReplacementPattern):
+    """
+    This transformation adds Convert to IR data_type after RandomUniform operation
```

Comment:
Please, explain in details *why* we insert Convert instead of modifying the `output_type` attribute value. Now it is not clear.

Reply:
Done.